src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -746,6 +748,15 @@ private void joinConferenceRoom()
         }
         catch (Exception e)
         {
+            if (e.getCause() instanceof XMPPException.XMPPErrorException)
+            {
+                if (((XMPPException.XMPPErrorException)e.getCause())
+                        .getXMPPError().getCondition() == service_unavailable)
+                {
+                    gatewaySession.notifyMaxOccupantsLimitReached();
```

Comment:
Non-blocking question/comment. Notifications are (at least in my mind..) handled quickly, but it seems here we block in order to handle the failure. If we want to handle the case where we reach the max occupants, why not just name the method `handleMaxOccupantsLimitReached`? Is it for consistency with the rest of the  method names?